src/Tizen.NUI.Components/Controls/ItemsView/CollectionView.cs

```diff
@@ -0,0 +1,815 @@
+        [EditorBrowsable(EditorBrowsableState.Never)]
+		public static readonly BindableProperty SelectedItemsProperty =
+			BindableProperty.Create(nameof(SelectedItems), typeof(IList<object>), typeof(CollectionView), null,
+				propertyChanged: (bindable, oldValue, newValue)=>
+                {
+                    var colView = (CollectionView)bindable;
+                    var oldSelection = (IList<object>)colView.selectedItems ?? s_empty;
+                    //FIXME : CoerceSelectedItems calls only isCreatedByXaml
+                    var newSelection = (SelectionList)CoerceSelectedItems(colView, newValue);
+                    colView.selectedItems = newSelection;
+			        colView.SelectedItemsPropertyChanged(oldSelection, newSelection);
+                },
+				defaultValueCreator: (bindable) =>
+                {
+                    var colView = (CollectionView)bindable;
+                    colView.selectedItems = colView.selectedItems ?? new SelectionList(colView);
+                    return colView.selectedItems;
+                });
+
+        private ViewItem header;
+
+        /// <summary>
+        /// Header item which placed in top-most position.
+        /// note : internal index and count will be increased.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ViewItem Header
+        {
+            get => header;
+            set
+            {
+                if (header != null)
+                {
+                    ContentContainer.Remove(header);
+                    header.Dispose();
+                }
+                if (value != null)
+                {
+                    value.Index = 0;
+                    value.ParentItemsView = this;
+                    value.IsHeader = true;
+                    ContentContainer.Add(value);
+                }
+                header = value;
+                Init();
+            }
+        }
+
+        private ViewItem footer;
+
+        /// <summary>
+        /// Footer item which placed in bottom-most position.
+        /// note : internal count will be increased.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ViewItem Footer
+        {
+            get => footer;
+            set
+            {
+                if (footer != null)
+                {
+                    ContentContainer.Remove(footer);
+                    footer.Dispose();             
+                }  
+                if (value != null)
+                {
+                    value.Index = InternalItemSource != null ? InternalItemSource.Count : 0;
```

Comment:
`??`를 이용해서 간단하게 줄일 수 있습니다.
```suggestion
                    value.Index = InternalItemSource?.Count ?? 0;
```